portal/src/app/components/home/components/experiment/components/toggles/Toggles.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import { ToggleButton } from "../../../../../../shared/components/toggle-button";
+import { EXPERIMENT_EN } from "../../translate/en";
+
+export interface TogglesProps {
+  currentSection: string;
+  handleButtonClick: (section: string) => void;
+}
+
+export const Toggles: React.FC<TogglesProps> = (props: TogglesProps) => {
```

Comment:
maybe try to rename the component to something more related , like `ExperimentTabs`